src/stactools/modis/fragments/MOD10A1/061/bands.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "name": "NDSI_Snow_Cover",
+        "description": " NDSI snow cover plus other results. This value is\n                            computed for MOD10_L2 and retrieved when the\n                            observation of the day is selected. Possible values\n                            are:\n                            0\u2013100: NDSI snow cover\n                            200: missing data\n                            201: no decision\n                            211: night\n                            237: inland water\n                            239: ocean\n                            250: cloud\n                            254: detector saturated\n                            255: fill"
```

Comment:
Looks like we still have some spurious newlines and quote characters here, so we should fix here and anywhere else in the PR.

Reply:
Yeah, my comment was from before we fixed things up, I think we're good meow.